bridges/otellogr/go.mod

```diff
@@ -1,19 +1,20 @@
 module go.opentelemetry.io/contrib/bridges/otellogr
 
-go 1.22
+go 1.22.0
```

Comment:
It looks like we are including the PATCH version of the go. Is this expected?

Reply:
It is a required change: https://github.com/open-telemetry/opentelemetry-go-contrib/compare/2108308cb125ac9c1d52ff50850e1c5746c888da..a6681a124544242c7e44d3930c0a5f906acd53f2

Prior to that CI fails. My understanding of this is based on https://github.com/open-telemetry/opentelemetry-go/blame/main/go.mod#L3, this is required now.